prometheus_es_exporter/__init__.py

```diff
@@ -354,11 +360,18 @@ def main():
         level=logging.DEBUG if args.verbose else log_level
     )
     logging.captureWarnings(True)
-
     port = args.port
     es_cluster = args.es_cluster.split(',')
 
-    if args.ca_certs:
+    if args.client_cert and args.client_key and args.ca_certs:
```

Comment:
It'd be good to add parser errors if the user provides bad combinations of these parameters, e.g. `client_cert` but no `client_key`, or `client_*` but not `ca_certs`, etc. Similar to the handling for `basic_user` and `basic_password`.